src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -57,7 +59,7 @@
  * use {@link ValueContainer#get(Key)} in order to retrieve information regarding
  * this item stack.</p>
  */
-public interface ItemStack extends SerializableDataHolder.Mutable, ComponentLike {
+public interface ItemStack extends SerializableDataHolder.Mutable, ComponentLike, HoverEventSource<HoverEvent.ShowItem> {
```

Comment:
Not that I'm expecting people to be implementing an ItemStack, but is there a way to default implement the method itself being added?

Reply:
Not in any complete manner -- the hover event includes SNBT-serialized data from the stack's tag, which we don't reeealy expose in API